student-os/scripts/materials_convert.py

```
@@ -722,6 +865,184 @@ def convert_with_mineru_chunked(
     return payload
 
 
+def effective_agent_chunk_pages(page_count: int, chunk_size: int) -> int:
```

Ruff (0.16.1)

[warning] Unused function argument: `page_count`

(ARG001)